DESCRIPTION

```diff
@@ -9,32 +9,30 @@ Authors@R: c(
     person("Laura", "Hilton", role = "ctb",
            comment = c(ORCID = "0000-0002-6413-6586")),
     person("Adam", "Mattsson", , "cmattsson@bcgsc.ca", role = "aut",
-           comment = c(ORCID = "0000-0002-6318-7912")),
-    person("Haya", "Shaalan", role = "ctb"),
-    person("Kristena", "Daley", role = "ctb")
+           comment = c(ORCID = "0000-0002-6318-7912"))
   )
 Description: Database, mining, manipulation and convenience functions for working with GAMBL results.
 License: MIT + file LICENSE
 Depends:
     R (>= 3.5.0)
 Imports:
     cli (>= 3.6.1),
-    GAMBLR.data (>= 1.1.1),
-    GAMBLR.helpers (>= 1.1.1),
-    GAMBLR.utils (>= 1.1.1),
-    GAMBLR.viz (>= 1.1.1),
-    GAMBLR.results (>= 1.1.1),
+    GAMBLR.data (>= 1.1),
```

Comment:
Should these be 1.2 to match remotes below?

Reply:
Yeah this is a tag id, not the version - so the one in remotes below should be 1 above what is mentioned here. If they match, the installation [complains about incorrect version](https://github.com/morinlab/GAMBLR/actions/runs/12957824528/job/36146940726#step:6:65). In the previous setup, this referred to 1.1.1 for each package and the remotes below were 1 tag above, 1.1.1a/b/c. So similar concept is preserved with the update. It does[ install the version tagged with 1.2](https://github.com/morinlab/GAMBLR/actions/runs/12958326646/job/36148503252#step:5:12).